src/main/java/org/ow2/proactive/addons/email/EmailSender.java

```diff
@@ -139,6 +139,8 @@
 
     protected String fileName = "attachment.txt";
 
+    protected Properties properties;
+
     protected EmailSender(boolean auth, boolean debug, boolean enableStartTls, int port, List<String> cc,
```

Comment:
EmailSender is built internally only. thus I would prefer to have a constructor with only explicit arguments and the rest in a Properties parameter.

something like:
`protected EmailSender(Properties properties, List<String> cc, List<String> bcc, List<String> recipients, String body, String from, String subject, String fileToAttach, String fileName)`

Note that `transport.connect(username, password);` should be replaced by `transport.connect();` as the authentication is already specified in the properties used to build the transport object

Reply:
Ok understood I will add a new constructor. Noted the `connect()` too

Reply:
I mean to replace the existing constructor